circle.yml

```diff
@@ -5,3 +5,7 @@ dependencies:
 test:
   override:
     - bundle exec rake test
+
+machine:
+  ruby:
+    version: 2.0.0
```

Comment:
I think you should run against newer Ruby, 2.3 or 2.4—despite 2.0.0 being the lowest required.